router/metrics/peekhandler.go

```diff
@@ -159,7 +160,7 @@ func NewXgressPeekHandler(registry metrics.UsageRegistry) xgress.PeekHandler {
 		egressTxMsgSizeHistogram:  egressTxMsgSizeHistogram,
 		egressRxMsgSizeHistogram:  egressRxMsgSizeHistogram,
 
```

Comment:
Can you add this to the config as well, please? Otherwise it's just a constant in a new place :)

Reply:
i'm not specifically sure what you're pointing at?